src/pages/text-curation/show/[datasetId].tsx

```diff
@@ -17,14 +24,21 @@ export const getServerSideProps = async (ctx) => {
   const { data } = await axShowDataset(ctx.query.datasetId);
 
   const user = getParsedUser(ctx);
+  const userName = user.name;
+  const canEdit =
+    data.contributors.includes(user.id.toString()) ||
+    hasAccess([Role.Admin], ctx) ||
+    data.validators.includes(user.id.toString());
 
-  const canEdit = data.contributors.includes(user.id.toString()) || hasAccess([Role.Admin], ctx);
+  const canValidate = data.validators.length == 0 || data.validators.includes(user.id.toString());
```

Comment:
admin check is removed from here, is that intentional?

Reply:
haven't removed admin access, just added the access to validators also